inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt

```diff
@@ -0,0 +1,22 @@
+defusedxml>=0.5.0
+scipy==1.2.1
+jstyleson==0.0.2
+numpy==1.16.3
+addict==2.2.1
+pandas==0.24.2
+hyperopt==0.1.2
+networkx==2.2
+tqdm==4.31.1
+py-cpuinfo==4.0.0
+PyYAML
+pillow>=2.6.1
```

Comment:
sometime ago, vulnerabilities was detected in Pillow, recommended to update on 8.1 and higher